### PR TITLE
Add heartbeat & timestamp info to alerts docs

### PR DESCRIPTION
**Pull request by nihalgonsalves**
Add a link to the event timestamp and SDK interval setup guide to provide more context on the start/end delay logic.

### CODE REVIEW

**Comment by nihalgonsalves** [comment]
Current dependencies on/for this PR:
* main
  * **PR #511** <a href="https://app.graphite.dev/github/pr/memfault/memfault-docs/511" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/memfault/memfault-docs/511?utm_source=stack-comment).

File: docs/platform/alerts.mdx [hunk 1]
```diff
@@ -159,6 +159,20 @@ interval of the device would behave similarly to setting no delay at all. A good
 rule of thumb to prevent noise is to use a delay, at minimum, slightly longer
 than the heartbeat interval.
 
+:::info
+
+By default, events are timestamped when they're uploaded to Memfault. This works
```
Comment (martijnthe) [inline docs/platform/alerts.mdx:164]: This is only true for MCU devices.

Linux & Android are timestamped on the device using the device's RTC when they are captured (RTC-less Linux/Android doesn't exist).

Comment (martijnthe) [inline docs/platform/alerts.mdx:164]:
```suggestion
By default, events captured by the MCU SDK are timestamped when they're uploaded to Memfault (on Linux and AOSP, events are always timestamped on-device using the device's RTC clock). This works
```

File: docs/platform/alerts.mdx [hunk 2]
```diff
@@ -159,6 +159,20 @@ interval of the device would behave similarly to setting no delay at all. A good
...
+well for frequently-connected devices, but for devices that batch-upload many
+heartbeats at once, this would result in all of the events in a batch being
+timestamped with the same timestamp. You can set an
+[**event timestamp**](../mcu/event-timestamps.mdx) using the SDK.
```
Comment (martijnthe) [inline docs/platform/alerts.mdx:168]:
```suggestion
[**event timestamp**](../mcu/event-timestamps.mdx) using the MCU SDK.
```

**Review by martijnthe — APPROVED** [review]
I'd make it explicit that this only applies to MCU and not Linux/AOSP. Otherwise, lgtm!

**Pull request merged**